src/integration-test/helpers/saucelabsReporter.js

```diff
@@ -12,7 +12,7 @@ function reportBuildResultToSaucelabs (result) {
   const sauceUsername = process.env.SAUCELABS_USERNAME
 
   request.put({
-    uri: `https://eu-central-1.saucelabs.com/rest/v1/${sauceUsername}/jobs/${sessionId}`,
+    uri: `https://eu-central-1.saucelabs.com/rest/v1/'+${sauceUsername}+'/jobs/'+${sessionId}`,
```

Comment:
this needs reverting - its not doing what you expect: https://jsfiddle.net/wa680vre/

or remove backticks and turn into a normal string